frontend/src/components/modules/Tournaments/EditTournament/EditInfo.tsx

```diff
@@ -0,0 +1,410 @@
+import React, { useState, useEffect } from "react";
+import { useNavigate, useParams } from "react-router-dom";
+import useToast from "hooks/useToast";
+import api from "api/axios";
+import ErrorModal from "components/common/ErrorModal";
+import routePaths from "routes/route-paths";
+import { useAuth } from "context/AuthContext";
+import type { Category, MatchTime, TournamentType } from "types/models";
+import { useTranslation } from "react-i18next";
+import {
+  CheckboxElement,
+  DateTimePickerElement,
+  FormContainer,
+  SelectElement,
+  TextFieldElement,
+  useForm,
+  useWatch
+} from "react-hook-form-mui";
+
+import {
+  Typography,
+  Button,
+  Container,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  Stack,
+  Box
+} from "@mui/material";
+
+import type { Dayjs } from "dayjs";
+import dayjs from "dayjs";
+import Loader from "components/common/Loader";
+const MIN_PLAYER_AMOUNT = 3;
+const now = dayjs();
+
+export interface EditTournamentFormData {
+  name: string;
+  location: string;
+  startDate: Dayjs;
+  endDate: Dayjs;
+  description: string;
+  type: TournamentType;
+  maxPlayers: number;
+  playersToPlayoffsPerGroup?: number;
+  groupsSizePreference?: number;
```

Comment:
Tässä datassa on tarvittavat arvot, mutta näitä ei voi ikinä muokata nyt. Eli siis uusia kenttiä ei tuu näkyviin kun valitsee esim. tietyn turnaustyypin, joka vaatii lisätietoja. Kantsii varmaan kattoo nää vaan suoraan siitä CreateTournament formista.

Reply:
Vissiinkin Melina olikin sanonu, että tota turnaustyyppiä ei sittenkään tarviis olla muokattavissa ollenkaan. Tähän vois tehä niin, että turnaustyyppiä ei siis voi muokata, mutta sen perusteella tulis noi lisäkentät (groupsSizePreference yms.) näkyviin ja niitä kaikkia vois muokata

Reply:
Nyt kävin puskeen muutokset, Eli turnaustyyppiä ei pysty vaihtaa muuta jos on pudotuspelit alkulohkoilla niin sitte näkyy myös lohkon max koko ja kuinka monta jatkoon. Ja joka turnaustyypille lisäks näkyy kenttien määrä.